### PR TITLE
mongo config - init

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- mongo DB config

### CODE REVIEW

File: app/helpers/api-util.js [hunk 1]
```diff
@@ -53,26 +50,26 @@ export async function getFeaturedTrips() {
 
 export async function getByIdTrips(id) {
   const all = await getAllTrips();
+  console.log(all);
```
Comment (doublethink13) [inline app/helpers/api-util.js:53]: remove console logs

File: app/helpers/api-util.js [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-  const response = await fetch(
-    "https://roads-blog-default-rtdb.firebaseio.com/roads/tours.json"
-  );
-  const data = await response.json();
+  try {
+    const client = await clientPromise;
+    const db = client.db("roads");
 
-  const NEW = [];
+    const allTrips = await db.collection("tours").find({}).toArray();
```
Comment (doublethink13) [inline app/helpers/api-util.js:9]: not trips

Comment (doublethink13) [reply app/helpers/api-util.js:9]: also check others below

File: app/helpers/api-util.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/helpers/api-util.js]: well noted! Thank you
Deleted

File: app/helpers/api-util.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/helpers/api-util.js]: my bad, thanks!!
Corrections done

**Pull request merged**